src/views/index.ejs

```diff
@@ -1,41 +1,50 @@
 <% include partials/header %>
-  <style>
-    .copy:hover {
-      color: #fff;
-    }
-  </style>
-  <div id="fb-root"></div>
-  <div class="wrapper">
-    <div class="container">
-      <!-- TRIM LOGO -->
-      <div class="row">
-        <div class="col-md-12 col-lg-12 col-sm-12">
-          <a href="/"> <img src="https://res.cloudinary.com/leankhan/image/upload/v1570985383/trim/trim-logo-2.png" alt="trim-logo" class="trim-logo" /></a>
-
-          <h1 class="big-heading">Shorten Your Links</h1>
-          <!-- INPUT URL -->
-          <div class="row">
-            <form action="/" method="POST">
-              <div class="input-group mb-3">
-                <input type="text" class="form-control form-control-lg" name="long_url" required />
-                <input type="hidden" name="created_by" value="<%=created_by%>">
-                <div class="input-group-append">
-                  <button type="submit" onclick="closeAlert()" class=" clip-btn btn btn-success"></button>
-                </div>
+<style>
+  footer {
+    padding: 20px;
+    margin-top: 150px;
+    background-color: rgb(20, 146, 58);
+    text-align: center;
+    color: #ffffff;
+    font-size: 20px;
+  }
```

Comment:
Is there a reason why we have this block here? I can see you already created a `main.css` file and there is a `style.css` file already.